WooCommerce/Classes/POS/ViewModels/PointOfSaleDashboardViewModel.swift

```diff
@@ -104,13 +104,12 @@ private extension PointOfSaleDashboardViewModel {
                 switch paymentState {
                 case .processingPayment,
                         .paymentError,
-                        .cardPaymentSuccessful:
-                    return true
-                case .idle,
-                        .acceptingCard,
+                        .cardPaymentSuccessful,
                         .validatingOrder,
                         .validatingOrderError,
```

Comment:
Let's allow coming back when `validatingOrderError`, otherwise users get stuck

Reply:
Yes, I see what you mean. Updated on https://github.com/woocommerce/woocommerce-ios/pull/13631/commits/30de5d60ca7c056c10ffc5a31d73d2e63a850f58

Reply:
Thanks!